src/runtime/server/server.ts

```diff
@@ -60,6 +61,13 @@ export function create(appState: AppState) {
           }) ?? noop
         )
       },
+      get executeGraphQL() {
```

Comment:
Executing a graphql query in its pure form seems like something that should be tied to the `schema` component, not the `server` component.